packages/x-date-pickers-pro/src/DateRangePickerDay/DateRangePickerDay.tsx

```diff
@@ -222,7 +252,14 @@ const DateRangePickerDayDay = styled(PickersDay, {
   }),
   ...(!ownerState.selected &&
     ownerState.isHighlighting && {
-      color: theme.palette.getContrastText(alpha(theme.palette.primary.light, 0.6)),
+      [`&.${dateRangePickerDayClasses.rangeIntervalOutsideCurrentMonth} .${dateRangePickerDayClasses.day}`]:
+        {
+          color: theme.palette.getContrastText(alpha(theme.palette.primary.light, 0.1)),
```

Comment:
So this is probably the line responsible for this change and it was intentional?
**Before:**
![Screenshot 2023-03-02 at 10 46 24](https://user-images.githubusercontent.com/4941090/222383234-32a47af5-f46b-486a-ac60-649c68a95ed3.png)

**After:**
<img width="300" alt="Screenshot 2023-03-02 at 10 46 49" src="https://user-images.githubusercontent.com/4941090/222383269-6d1e0913-d853-4f23-93ec-8d81efdea33e.png">

Reply:
Finally got it. I thought `&` was referring to the div wrapping the entire day, but it's referring to the button